colorama/initialise.py

```diff
@@ -22,23 +22,26 @@ def reset_all():
 
 def init(autoreset=False, convert=None, strip=None, wrap=True):
 
+    if wrap not in [True, False, "stdout", "stderr"]:
+        raise ValueError('wrap must be one of: True, False, stdout, stderr')
+
     if not wrap and any([autoreset, convert, strip]):
         raise ValueError('wrap=False conflicts with any other arg=True')
 
     global wrapped_stdout, wrapped_stderr
     global orig_stdout, orig_stderr
 
-    orig_stdout = sys.stdout
-    orig_stderr = sys.stderr
-
     if sys.stdout is None:
         wrapped_stdout = None
-    else:
+    elif wrap != "stderr":
```

Comment:
Forgive my very overdue drive-by review, but does this do the wrong thing if wrap==False ?